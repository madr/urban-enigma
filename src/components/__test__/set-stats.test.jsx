import React from "react";
import renderer from "react-test-renderer";
import SetStats from "../set-stats";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <SetStats sets={[{ weight: 100, reps: 5 }, { weight: 80, reps: 8 }]} />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
