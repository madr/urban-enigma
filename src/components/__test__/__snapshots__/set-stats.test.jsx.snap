// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul>
  <li>
    <b>
      1.14
    </b>
     ton
  </li>
  <li>
    <b>
      2
    </b>
     set
  </li>
  <li>
    <b>
      13
    </b>
     reps
  </li>
</ul>
`;
